cms/envs/common.py

```diff
@@ -944,6 +944,10 @@
 
     # management of user-triggered async tasks (course import/export, etc.)
     'user_tasks',
+
+    # CMS specific user task handling
+    'cms_user_tasks',
```

Comment:
Yep, changing this to reference `CmsUserTasksConfig` as that seems to be best practice.